skbio/sequence/tests/test_distance.py

```diff
@@ -126,5 +126,76 @@ def test_sequences_with_metadata(self):
             self.assertEqual(distance, 0.75)
 
 
+class TestKmerDistance(unittest.TestCase):
```

Comment:
Can you add the following tests?

- [ ] compute distance between two sequences that aren't the same length
- [ ] compute distance between two sequences of length zero (should return `np.nan`)
- [ ] compute distance between a length zero sequence and non-empty sequence (should return 1.0)
- [ ] compute distance where there are no kmers (i.e., passing a value of `k` that is too large), should return `np.nan`
- [ ] compute distance between two subclasses of `Sequence` (e.g., `DNA`)
- [ ] add test similar to Hamming's `test_return_type`